### ISSUE TITLE
MiAIRR elements from staging

### ISSUE THREAD

**Issue by graybeal** [metadatacenter/cedar-project#1051]
Looking at the [template for MiAIRR v1.1](https://cedar.metadatacenter.org/templates/edit/https://repo.metadatacenter.org/templates/ea716306-5263-4f7a-9155-b7958f566933) I saw a number of identifiers that included staging, e.g. 
```
* https://repo.staging.metadatacenter.net/template-fields/f79fbd3d-cb7f-4837-8167-b890a9ed96fd (Other Relevant Medical History)
* https://repo.staging.metadatacenter.net/template-fields/44e3ff78-e8f6-4892-bf9e-dc1265ea2012
(Subject)
* "Contact Information (data collection)": {
              "enum": [
                "https://schema.staging.metadatacenter.org/properties/d4210479-a917-4af6-aee4-c51ee39ec8fd"
*              "Cell Isolation": {
                "enum": [
                  "https://schema.staging.metadatacenter.net/properties/cellIsolation"
*  "pav:derivedFrom": "https://repo.staging.metadatacenter.org/templates/ade1336d-a292-49f8-8fb9-d62c11997820".
```

These appear to be coming from the elements, all of which have these throughout.

The question is, what should be done about these?

**Comment by graybeal** [comment]
While originally I wanted to clean these up, I think there are two reasons not to do so now, at least for the @ids.  First, it is not essential, as the identifiers are not resolvable currently, they are just strings. Second, it would be a lot of work; at a minimum all 3 elements have to be corrected, then re-imported. There isn't much chance of error in such a change, so perhaps it could be performed when a new version is published.

What about the derivedFrom entities? Should we fix to point back to the predecessors on the production system, which I think exist?

**Comment by marcosmro** [comment]
I agree. We can fix those @ids when we update the template. I also agree with your suggestion about the derivedFrom references. The entities in production are derived from entities in staging, so I think it's accurate to keep those references as they are.
